src/components/Events/Events.js

```diff
@@ -155,6 +156,16 @@ class Events extends Component {
       });
   }
 
+  cleanSearchText(filterData) {
+    // removes apostrophes from user input to avoid broken searches
+    // without affecting what the user sees in the search box
+    const filtersClean = Object.assign({}, filterData);
+    if (filtersClean.searchText) {
+      filtersClean.searchText = filtersClean.searchText.replace(/'/g, '');
```

Comment:
It seems like a search of `Trump's Agenda` would query `Trumps Agenda` and thus not return `March to Stop Trump's Agenda`

Reply:
@pdw207 - you bring up a good point. Thing is, it was indeed getting encoded properly as `%27` upon being POST'ed to the API - so maybe @aaghevli can weigh in as to whether or not this is something that should be handled on the back end, since it looks like it was being submitted correctly as is, but the back end was returning a 500. Whaddyafink?

Reply:
Agree, I think we should fix it in the backend.